### PR TITLE
[NIFI-12763] - Theme mat paginator - unset the background color so it blends in with the theme

### PR DESCRIPTION
**Pull request by rfellows**

### CODE REVIEW

**Comment by scottyaslan** [comment]
Reviewing...

File: nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/styles/_app.scss [hunk 1]
```diff
@@ -136,6 +136,10 @@
         flex-basis: 100%;
         flex: 1;
     }
+
+    .mat-mdc-paginator {
+        background-color: unset !important;
```
Comment (scottyaslan) [inline nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/styles/_app.scss:141]: I think we likely want to just set it to the mat-app-background-color:

```suggestion
        background-color: var(--mat-app-background-color) !important;
```

**Review by scottyaslan — CHANGES_REQUESTED** [review]
Thanks @rfellows this is a good improvement! I left a minor comment below.

File: nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/styles/_app.scss (hunk 1 above)
Comment (rfellows) [reply nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/styles/_app.scss:141]: That assumes we always want to put the paginator on our app background. setting is as `unset` allows it to be used on any background.

Comment (scottyaslan) [reply nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/styles/_app.scss:141]: Fair enough. Let's roll.

**Pull request merged**